package.json

```diff
@@ -30,13 +30,16 @@
     "clean:install": "npm run clean && npm install",
     "danger": "danger ci",
     "lint": "eslint --ext .js,.jsx .",
+    "test": "tape lib/**/__tests__/**/*.js",
```

Comment:
Is keeping tests in lib a stylelint convention?

Reply:
Yes, tests, documentation, and source code are structured until lib/rules/rule in stylelint.

Here is an example: https://github.com/stylelint/stylelint/tree/master/lib/rules/at-rule-blacklist

I'm not strongly opinionated if we want to restructure this design.

Reply:
I'm not either, just curious since I haven't done any stylelint plugin / rule development. That being said, should we add `lib/**/__tests__/` to the npm ignore so we don't ship tests?

Reply:
We definitely should not ship these upon release, I'll update the npm ignore when we've landed on where the tests will live

Reply:
Lets keep them here since it aligns with stylelint plugin standards